mathics/builtin/arithfns/basic.py

```diff
@@ -289,22 +289,17 @@ def format_plus(self, items, evaluation):
         "Plus[items__]"
 
         def negate(item):
-            if item.has_form("Times", 1, None):
```

Comment:
Why is it this we know that line can be removed?

Reply:
If it already is `Times`, we are evolving its leaves in another `Times`, I'll attach the benchmarks.

Reply:
And if it isn't, we are evolving in `Times` too, so same thing.